sky/backends/cloud_vm_ray_backend.py

```diff
@@ -3115,6 +3159,22 @@ def teardown_no_lock(self,
                 # successfully removed cluster as no exception was raised
                 returncode = 0
 
+        # Apr, 2023 by Hysun(hysun.he@oracle.com): Added support for OCI
+        # May, 2023 by Hysun: Allow terminate INIT cluster which may have
+        # some instances provisioning in backgroud but not completed.
+        elif (isinstance(cloud, clouds.OCI) and terminate and
+              prev_cluster_status in (global_user_state.ClusterStatus.STOPPED,
+                                      global_user_state.ClusterStatus.INIT)):
+            region = config['provider']['region']
+
+            # pylint: disable=import-outside-toplevel
+            from sky.skylet.providers.oci.query_helper import oci_query_helper
+            from ray.autoscaler.tags import TAG_RAY_CLUSTER_NAME
+
+            # 0: All terminated successfully, failed count otherwise
+            returncode = oci_query_helper.terminate_instances_by_tags(
+                {TAG_RAY_CLUSTER_NAME: cluster_name}, region)
```

Comment:
Do we have the `stdout` and `stderr` here? If not, we may need to manually fill in some stdout/stderr (or assign them with empty strings), to make it easier to find out the reason for the failure. (just a note: the stdout and stderr will be used in L3269 and printed out.)

Reply:
Added traceback.print_exc() to print stacktrace to stderr if exception happened.

Reply:
Sounds good. Please add the assignment for the `stdout` and `stderr` to avoid undefined local variable later.
```
stdout = stderr = ''
```